src/Build/Microsoft.Build.csproj

```diff
@@ -1,6 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(RuntimeOutputTargetFrameworks)</TargetFrameworks>
+    <!-- Node reuse requires an API new to .NET Core 2.1 not yet available
+         in .NETStandard. -->
+    <TargetFrameworks>netstandard2.0;netcoreapp2.1</TargetFrameworks>
```

Comment:
How does this not break our netcoreapp2.0 target? I assume that one's also getting tested, right?

Reply:
The netcoreapp2.0 target uses the netstandard2.0 library from here.